app/src/main/java/ch/epfl/sweng/swenggolf/database/CreateUserActivity.java

```diff
@@ -65,15 +66,14 @@ public void onClick(View view) {
         String userName = name.getText().toString();
         String userMail = mail.getText().toString();
 
-        //handle the exception if the EditText fields are null
+        // Handle the exception if the EditText fields are null
         if (!userName.isEmpty() && !userMail.isEmpty()) {
```

Comment:
Should we verify that the mail has correct format? eg: palpatin@do.it

Reply:
Good idea.